lib/parser/init-transform.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+const path = require('path');
+const transform = require('./transformations').transform;
+
+module.exports = function initTransform(opts) {
+	global.options = opts;
+	const templateFP = path.join(__dirname, 'utils', 'template.js');
+	const template = fs.readFileSync(templateFP, 'utf8');
+	const ast = transform(template);
```

Comment:
Is the initial work, I want @pksjce to work on this, I'm not fluent enough in writing AST's yet.